src/gmusic-theme.js

```diff
@@ -132,13 +160,25 @@ window.GMusicTheme = class GMusicTheme {
   }
 
   _substituteColors(styleString) {
+    // DEV: If replacing all colors
+    if (this.enabled === 1) {
+      return styleString
+        .replace(/<<BACK_PRIMARY>>/g, this.BACK_PRIMARY)
+        .replace(/<<BACK_SECONDARY>>/g, this.BACK_SECONDARY)
+        .replace(/<<BACK_HIGHLIGHT>>/g, this.BACK_HIGHLIGHT)
+        .replace(/<<FORE_PRIMARY>>/g, this.FORE_PRIMARY)
+        .replace(/<<FORE_SECONDARY>>/g, this.FORE_SECONDARY)
+        .replace(/<<BACK_SECONDARY_O>>/g, this._rgba(this.BACK_SECONDARY, 0.5))
+        .replace(/<<NOTIMPORTANT>> \!important/g, '');
+    }
+    // DEV: Else remove all rules for anything that isn't the highlight color (foreSecondary)
     return styleString
-      .replace(/<<BACK_PRIMARY>>/g, this.BACK_PRIMARY)
-      .replace(/<<BACK_SECONDARY>>/g, this.BACK_SECONDARY)
-      .replace(/<<BACK_HIGHLIGHT>>/g, this.BACK_HIGHLIGHT)
-      .replace(/<<FORE_PRIMARY>>/g, this.FORE_PRIMARY)
+      .replace(/\n.+<<BACK_PRIMARY>>.*;\n/g, '')
```

Comment:
`.+` and `.*` is definitely something to avoid using; it's a very greedy and costly regexp. Instead, use something more targetted like `[^;]+`. That being said, why do we have these new matchers?